mmdet3d/models/backbones/pointnet2_sa_msg.py

```diff
@@ -105,7 +105,7 @@ def __init__(self,
                     dilated_group=dilated_group[sa_index],
                     norm_cfg=norm_cfg,
                     cfg=sa_cfg,
-                    bias=True))
+                    bias=False))
```

Comment:
Does this change affect other models' performance?

Reply:
I have converted pcdet's PointRCNN checkpoints to mmdet3d, and I find here is different with pcdet lead to param num excess. I will fix it and test it again.